src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -74,6 +75,7 @@ import org.springframework.security.core.userdetails.UserDetails
  * will be cached afterwards.
  */
 data class IndividualUser(
+    val createdTime: Instant?,
```

Comment:
I believe this can be non-nullable since the underlying column in the users table is.

Reply:
That is what I originally tried, but the `UsersRow` is nullable. `build/generated-jooq/com/terraformation/backend/db/default_schema/tables/pojos/UsersRow.kt:29`

![SCR-20240315-jsvj.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/wJnPX9wLZAkiGXEBFJxL/db83b3cb-562e-431a-a1c4-e4913fd4d547.png)

Perhaps I need to use the `usersRow.createdTime!!` when creating `IndividualUser`?